CodeGeneration/Sources/generate-swift-syntax/templates/swiftsyntax/SyntaxRewriterFile.swift

```diff
@@ -101,10 +101,14 @@ let syntaxRewriterFile = SourceFileSyntax(leadingTrivia: copyrightHeader) {
       /// Override point to choose custom visitation dispatch instead of the
       /// specialized `visit(_:)` methods. Use this instead of those methods if
       /// you intend to dynamically dispatch rewriting behavior.
-      /// - note: If this method returns a non-nil result, the specialized
-      ///         `visit(_:)` methods will not be called for this node and the
+      /// - note: If this method returns a non-nil result, the subsequent
+      ///         `visitAny(_:)` methods and the specialized `visit(_:)`
+      ///         methods will not be called for this node and the
       ///         visited node will be replaced by the returned node in the
       ///         rewritten tree.
+      ///         You can call the `SyntaxWriter.rewrite(_:detach:)`
```

Comment:
You should be able to use the `/` syntax to create a docc link. Also noticed that there’s a typo `SyntaxWriter` -> `SytnaxRewriter`

```suggestion
      ///         You can call the ``SyntaxRewriter/rewrite(_:detach:)``
```